test/helpers/wrappers.go

```diff
@@ -104,6 +104,24 @@ func CurlWithHTTPCode(endpoint string, optionalValues ...interface{}) string {
 		CurlConnectTimeout, endpoint)
 }
 
+// CurlWithRetries returns the string representation of the curl command that
+// retry the request if transient problems occur. Parameter retries are indicated the maximum retry times.
+// If flag "fail" is true, the function will called CurlFail() and add --retry flag at the end of the command and return.
```

Comment:
Nit:
```suggestion
// If flag "fail" is true, the function will call CurlFail() and add --retry flag at the end of the command and return.
```

Reply:
Thanks! I've updated the PR. Please take a look.